app/apps/cases/serializers/case.py

```diff
@@ -147,18 +147,19 @@ class CaseCreateSerializer(BaseCaseSerializer, WritableNestedModelSerializer):
     )
     subjects = SubjectSerializer(many=True, read_only=True)
     subject_ids = serializers.PrimaryKeyRelatedField(
+        source="subjects",
         required=False,
         many=True,
         write_only=True,
         queryset=Subject.objects.all(),
-        source="subjects",
     )
-    tag = TagSerializer(read_only=True)
-    tag_id = serializers.PrimaryKeyRelatedField(
-        source="tag",
+    tags = TagSerializer(many=True, read_only=True)
+    tag_ids = serializers.PrimaryKeyRelatedField(
```

Comment:
Dit ziet er ergens dubbelop uit. Is het nodig om tag_ids ook te serializen als je tags ook al serializeert?

Reply:
Jazeker, door `tag_ids` te "serializen", ontstaat de relatie en wordt er gebruik gemaakt van het DJANGO REST framework. Een binnenkomende `tag_id` wordt hierdoor gevalideerd.